src/apps/shared/test/unit/test_shared_custom_validation.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from apps.shared.domain.custom_validations import sanitize_string
+
+
+@pytest.mark.parametrize(
+    "value, sanitized_value_expected",
+    (
+        # needs sanitization
+        ("One <script>alert('Injected!')</script> Two", "One  Two"),
+        ("<script>alert('Injected!');</script>", ""),
+        ("""<img src="nothing" onerror="alert('Injected!');">""", '<img src="nothing">'),
+        ("""<div style="background-image: url('javascript:alert(\'Injected!\')')">""", "<div></div>"),
+        ("""<a href="javascript:alert('Injected!')">Click me</a>""", "<a>Click me</a>"),
+        ("""<input type="text" value="<img src=x onerror=alert('Injected!')>">""", ""),
+        # don't need sanitization
```

Comment:
Please add more valid cases, including equivalents of invalid ones above, to ensure images, links, etc work fine